internal/unionstore/pipelined_memdb.go

```diff
@@ -174,6 +176,62 @@ func (p *PipelinedMemDB) GetFlags(k []byte) (kv.KeyFlags, error) {
 	return f, nil
 }
 
+func (p *PipelinedMemDB) Prefetch(ctx context.Context, keys [][]byte) (map[string][]byte, error) {
+	m := make(map[string][]byte, len(keys))
+	if p.prefetchCache == nil {
+		p.prefetchCache = make(map[string]util.Option[[]byte], len(keys))
+	}
+	shrinkKeys := make([][]byte, 0, len(keys))
+	for _, k := range keys {
+		v, err := p.get(ctx, k, true)
+		if err != nil {
+			if tikverr.IsErrNotFound(err) {
+				shrinkKeys = append(shrinkKeys, k)
+				continue
+			}
+			return nil, err
+		}
+		if len(v) > 0 {
+			m[string(k)] = v
+			p.prefetchCache[string(k)] = util.Some(v)
+		} else {
+			p.prefetchCache[string(k)] = util.None[[]byte]()
```

Comment:
```suggestion
			p.prefetchCache[string(k)] = util.Some(v)
```
err == nil, so this is an empty value, right?
Should it be like this?